docs/components/connectors/use-connectors/inbound.md

```diff
@@ -87,9 +119,67 @@ For example, given that your correlation key is defined with `requestIdValue` pr
 
 See the [webhook documentation](/components/connectors/protocol/http-webhook.md) or the documentation of [other Connector types](/components/connectors/out-of-the-box-connectors/available-connectors-overview.md) for more details.
 
-### Working with request context
+## Connector deduplication
+
+In the simplest case, each inbound Connector element in a BPMN diagram corresponds to a unique endpoint, event consumer, or a polling task.
+However, sometimes you might want to have multiple BPMN elements listening to the same event source. For example, you might want to link multiple Connector events to the same message queue consumer and activate only one of them based on the message content.
+
+Consider the following BPMN diagram:
+
+![Connector deduplication use-case example](../img/deduplication-example.png)
+
+In this diagram, two Connector events are listening to the same message queue that can contain messages of two types: `PAYMENT_COMPLETED` and `PAYMENT_CANCELLED`.
+When the process execution arrives at the event gateway, the type of the message determines which path the process will take.
+If each Connector event listened to the message queue using a separate subscription, this might lead to race conditions due to message being received by a different consumer (e.g. `PAYMENT_COMPLETED` event being consumed by the consumer that expects `PAYMENT_CANCELLED`).
+Eventually this might lead to message loss or delayed processing, while also increasing the load on the message broker due to message being returned to the queue.
+
+To avoid this, both events can be assigned to the same subscription by assigning the same **Deduplication ID** to both events. Then, all messages will be consumed by the same subscription, and the Connector runtime will evaluate the **Activation condition** of each event to determine which one should be triggered.
+
+:::note
+When using this pattern, make sure that the **Activation condition** of each event is mutually exclusive, so that only one event is triggered for each message.
+Attempting to trigger multiple events for the same message will result in an error.
+:::
+
+### Automatic deduplication
+
+By default, the Connector runtime will assign the same deduplication ID to Connector events that have equal properties, and different deduplication IDs to events that have different properties.
```

Comment:
We might also want to mention that equality exactly the same values including whitespaces.